lib/ansible/modules/web_infrastructure/sentry_project.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Mikhail Naletov <okgolove@markeloff.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: sentry_project
+short_description: Manage Sentry project
+description:
+    - Manage the state of a project in Sentry.
+version_added: "2.8"
+
+options:
+  api_token:
+    description:
+      - API token generated in Sentry. The token has to have an acess for manipulating projects.
+    required: true
+
+  organization:
+    description:
+      - Organization containing a project.
+    required: true
+
+  project_name:
+    description:
+      - Project name you want to work with.
+    required: true
+
+  project_slug:
+    description:
+      - Slug name. Slug name must be uniq (even after project deletion you can't create a project with same slug name).
+    required: true
+
+  state:
+    description:
+      - The state of project.
+    required: true
+    choices: [ "present", "absent" ]
+
+  team:
+    description:
+      - A team you are in and which has an access for project manipulating.
+
+  url:
+    description:
+      - Sentry URL with scheme.
```

Comment:
You could use urlparse to automatically default the scheme to https (sane standards). In addition, it would make sense to have this as the default to "sentry.io", which is Sentry's hosted environment. This allows users to set it to their self-hosted solutions still if needed.

Reply:
I've used urlparse, but I wasn't able to use get_url() function because it returns url with 3 slashes https://stackoverflow.com/questions/7289481/urlparse-urlparse-returning-3-instead-of-2-after-scheme